### PR TITLE
Add lookback blocks to off-chain config, and add configurable offchain-config items to the script

### PR DESCRIPTION
**Pull request by vreff**
Adds lookback blocks to the Coordinator Config in the VRF offchain config, and add the coordinator config items to the ocr2vrf script configurable flags. Also removes unused confirmation delay flag that is [not used in the plugin config](https://github.com/smartcontractkit/chainlink/blob/chore/lookback-blocks/core/services/ocr2/plugins/ocr2vrf/config/config.go#L18-L28).

OCR2VRF Commit: https://github.com/smartcontractkit/ocr2vrf/pull/23

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -1586,8 +1586,8 @@ github.com/smartcontractkit/libocr v0.0.0-20220812191430-db92a9fdaa52 h1:Nac1UCK
 github.com/smartcontractkit/libocr v0.0.0-20220812191430-db92a9fdaa52/go.mod h1:5JnCHuYgmIP9ZyXzgAfI5Iwu0WxBtBKp+ApeT5o1Cjw=
 github.com/smartcontractkit/ocr2keepers v0.4.8 h1:pvCDWIsYaSJI/M6qw0xIIXpHs5MPQaYEpmWsJiyi6LM=
 github.com/smartcontractkit/ocr2keepers v0.4.8/go.mod h1:6wSlAWYBmlJM7bDZ4Ewc5J5AJQszIRGDNaUAw35r3qc=
-github.com/smartcontractkit/ocr2vrf v0.0.0-20221103213748-db2f67990850 h1:cZX0weC3QmHlS+stmLaliNgd4GfemmEl3tziZSYXKk8=
-github.com/smartcontractkit/ocr2vrf v0.0.0-20221103213748-db2f67990850/go.mod h1:NKkp8yf3trq+hJe/gn2Q1dd4abZvcYavUKPzYJCgVew=
+github.com/smartcontractkit/ocr2vrf v0.0.0-20221108202008-9139c7142460 h1:8oERPLVYE/F8rtBoo8IRqgx7MjuvNh2TNShq8dL0PF8=
+github.com/smartcontractkit/ocr2vrf v0.0.0-20221108202008-9139c7142460/go.mod h1:NKkp8yf3trq+hJe/gn2Q1dd4abZvcYavUKPzYJCgVew=
```
Comment (makramkd) [inline go.sum:1590]: LGTM - probably need to update this after merging ocr2vrf

Comment (vreff) [reply go.sum:1590]: yup yup

File: core/services/ocr2/plugins/ocr2vrf/internal/ocr2vrf_integration_test.go [hunk 2]
```diff
@@ -447,7 +446,6 @@ lookbackBlocks         	= %d # This is an integer
 			uni.coordinatorAddress.String(),
 			uni.feedAddress.String(),
 			"[1, 2, 3, 4, 5, 6, 7, 8]", // conf delays
```
Comment (makramkd) [inline core/services/ocr2/plugins/ocr2vrf/internal/ocr2vrf_integration_test.go:448]: And this line
```suggestion
```

File: core/services/ocr2/plugins/ocr2vrf/internal/ocr2vrf_integration_test.go [hunk 3]
```diff
@@ -435,7 +435,6 @@ dkgContractAddress     	= "%s"
 vrfCoordinatorAddress   = "%s"
 linkEthFeedAddress     	= "%s"
 confirmationDelays     	= %s # This is an array
```
Comment (makramkd) [inline core/services/ocr2/plugins/ocr2vrf/internal/ocr2vrf_integration_test.go:437]: Can remove this line.

**Review by makramkd — APPROVED** [review]
:shipit:

**Pull request merged**